artifacts/Music/source/PlayRecordRenderer.js

```diff
@@ -40,6 +40,7 @@ defineParticle(({DomParticle, html}) => {
     }
     render({playRecord}) {
       return {
+        subId: playRecord.id, // For rendering to a set slot.
```

Comment:
sorry, i'm not sure i understand why can't the CollectionRenderer be adding the subIds to the hosted model, same as the current multiplexer does?

Reply:
From my discussion with @shans we don't want the multiplexer to intercept rendering to slots other than the hosted slot it created. Here, the List particle writes to the hosted slot, so the transformation can do whatever it wants with the template and model of the what List renders, but what happens between List and the hosted particle doing the rendering is not visible to the transformation.

I just pushed a patch that adds a funny intermediate particle that renders a subId as model, and it 'fixed it', but now I'm wondering whether the slot between the List and the Renderers should also be hosted, and therefore intercepted? I'll think more about how that would look like tomorrow, it's late today already.

Thanks for asking and making me think.

Reply:
I'm not sure I understand - what is `transformation` vs `transformation particle` (which is List)?

Reply:
The way I've been using these terms is:
_transformation particle_ - a particle creating an inner arc and governing it.
_transformation_ - it is an inner arc + mappings between the inner and the outer arc, created and governed by the transformation particle.

@shans is that consistent with how you use these terms?

`CollectionRenderer` is a transformation particle here. `List` and `PlayRecordRenderer` are regular particles that are instantiated inside a transformation, but they are not transformation particles, as they don't create their own inner arcs.

Reply:
Not completely consistent, but reasonable concepts. I think I've been using transformation as the combination of what you call transformation and transformation particle.